### PR TITLE
Load in the rugby scores 

### PR DESCRIPTION
**Pull request by jennysivapalan**
- An A/B test that has zero audience (so opt-in only) that loads the scores for a rugby match on the live blog or match report or live blog.
- Currently lists teams playing this weekend (I will add the rest of the teams next week)

This is how it currently looks (not great) but we want to check how often this updates over the game this weekend so will add fixes to styling next week:
screen shot 2015-08-21 at 15 37 35

### CODE REVIEW

File: common/app/model/meta.scala [hunk 1]
```diff
@@ -352,6 +353,9 @@ trait Tags {
     tags.exists(t => t.id == "sport/england-cricket-team") &&
     tags.exists(t => t.id == "sport/over-by-over-reports")
 
+  lazy val isRugbyMatch = (isMatchReport || isLiveBlog) &&
```
Comment (janua) [inline common/app/model/meta.scala]: Maybe `isLiveRugbyMatch` ?

File: static/src/javascripts/bootstraps/sport.js [hunk 2]
```diff
@@ -20,7 +24,36 @@ define([
...
+
+                    $h.after(scoreContainer);
+                }
+            );
+
+        }
+    }
+
+    function init()  {
+        cricket();
```
Comment (stephanfowler) [inline static/src/javascripts/bootstraps/sport.js]: Do you need this intermediate fn?

File: common/app/model/meta.scala (hunk 1 above)
Comment (janua) [inline common/app/model/meta.scala]: No wait, `isRugbyMatchLiveBlog`, because there is also `liveBloggingNow`.

**Comment by stephanfowler** [comment]
:+1: for js

File: common/app/model/content.scala [hunk 3]
```diff
@@ -605,13 +606,21 @@ class LiveBlog(content: ApiContentWithMeta) extends Article(content) {
...
   )
 
-  private lazy val cricketMetaData = if (isCricketLiveBlog && conf.Switches.CricketScoresSwitch.isSwitchedOn) {
-    Map(("cricketMatch", JsString(webPublicationDate.withZone(DateTimeZone.UTC).toString("yyyy-MM-dd"))))
-  } else {
-    Map()
+  private lazy val sportMetaData = {
+    val rugbyMeta = if (isRugbyMatch) {
+      val teamIds = keywords.map(_.id).collect(RugbyContent.teamNameIds)
+      val (team1, team2) = (teamIds.headOption.getOrElse(""), teamIds.lift(1).getOrElse(""))
```
Comment (janua) [inline common/app/model/content.scala]: Would a `match` be nicer here?
```
teamIds match {
 case team1 :: team2 =>
 case _
}
```

**Comment by janua** [comment]
:+1: scala

File: common/app/model/meta.scala (hunk 1 above)
Comment (jennysivapalan) [inline common/app/model/meta.scala]: We want this on match reports so `live` seems wrong here.

**Comment by johnduffell** [comment]
I love the team work on this PR!

**Comment by jennysivapalan** [comment]
@johnduffell :man: :rugby_football: :woman: :runner: :trophy: :earth_americas: :rose:

**Comment by johnduffell** [comment]
:joy: :laughing:  haha it is rugby so I should have expected it ;)

**Pull request merged**